packages/eslint-config-loris/react.js

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+    plugins: ['react'],
+
+    parserOptions: {
+        ecmaFeatures: {
+            jsx: true
+        }
+    },
+
+    rules: {
+        // http://eslint.org/docs/rules/#stylistic-issues
+        'jsx-quotes': ['error', 'prefer-double'],
+
+        // https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules
+        'react/default-props-match-prop-types': 'off',
+        'react/display-name': 'off',
+        'react/forbid-component-props': 'off',
+        'react/forbid-elements': 'off',
+        'react/forbid-prop-types': 'off',
+        'react/forbid-foreign-prop-types': 'off',
+        'react/no-array-index-key': 'off',
+        'react/no-children-prop': 'off',
+        'react/no-danger': 'off',
+        'react/no-danger-with-children': 'error',
+        'react/no-deprecated': 'error',
+        'react/no-did-mount-set-state': 'error',
+        'react/no-did-update-set-state': 'error',
+        'react/no-direct-mutation-state': 'error',
+        'react/no-find-dom-node': 'off',
+        'react/no-is-mounted': 'error',
+        'react/no-multi-comp': 'warn',
```

Comment:
This is the only warning in the whole preset. I suggest that we use either 'off' or 'error'.